### ISSUE TITLE
Update installation instructions for proxy environment

### ISSUE THREAD

**Issue by lokeshh** [Nithmr/ProxyHelper#32]
This app is working wonderfully. I've tried it in computer lab. It avoids all the hassle of moving around and setting proxy everywhere and one is ready to go with very less effort.

One problem though I think a beginner would face will be how to install this app in a proxy environment because the instructions mention use of `git clone...` but it won't work in a proxy environment on its own.

**Comment by clearnote01** [comment]
Yeah! I have been working to solve this problem for sometime. It was a long process to migrate that "install in ~/.proxyhelper directory" to "install from anywhere" workflow. What makes this wonderful is you can simply download the zip from github link and run the install.sh file. No need for complex instructions in README.

**Comment by clearnote01** [comment]
#39 solves this problem.

Still the README needs to be updated to reflect this change.
